### DISCUSSION TITLE
Is there an installer for MacOS?

### DISCUSSION THREAD

**Discussion by StevenOrikami** [git-lfs/git-lfs#5192] [General]
Is there an installer for MacOS? The homepage only has windows.

Please take this down if its in the wrong spot! Thanks.

**Comment by bk2204** [comment]
We don't have an installer for macOS.  There is a zip file with notarized binaries which can be installed anywhere in your `PATH`.  Alternatively, you can use Homebrew, which also is a fine choice.
